components/mail/dovecot/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		dovecot
+COMPONENT_VERSION=	2.2.25
+COMPONENT_SUMMARY=	Dovecot is an open source IMAP and POP3 email server for Linux/UNIX-like systems.
+COMPONENT_PROJECT_URL=	http://dovecot.org
+COMPONENT_FMRI=		service/network/imap/dovecot
+COMPONENT_CLASSIFICATION= System/Services
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=	http://dovecot.org/releases/2.2/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	sha256:d8d9f32c846397f7c22749a84c5cf6f59c55ff7ded3dc9f07749a255182f9667
+COMPONENT_LICENSE=	MIT/LGPL2.1
+COMPONENT_LICENSE_FILE=	COPYING
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+CONFIGURE_OPTIONS += --sysconfdir=/etc
+CONFIGURE_OPTIONS += --with-ioloop=poll
+CONFIGURE_OPTIONS += --with-sqlite
+CONFIGURE_OPTIONS += --with-sql=yes
```

Comment:
I tried enabling ldap, but the build failed. I don't really want to spend the time right now to figure out why, so I'm going to leave ldap support off.